rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+
+    @book = book
+    @book.rentals << self
+
+    @person = person
```

Comment:
- [ ] The rental should be adding itself to the `@person` the `belongs-to` association as you did for the book above, kindly fix that 😉

Reply:
also, done, thank you for your review